### ISSUE TITLE
Error: The terraform-provider-snowflake_v0.80.0 plugin crashed

### ISSUE THREAD

**Issue by jacobcbeaudin** [Snowflake-Labs/terraform-provider-snowflake#2283]
### Terraform CLI and Provider Versions

Terraform CLI version: 1.6.1
Snowflake Provider Version: 0.80.0

### Terraform Configuration

```terraform
terraform {

  required_version = ">= 1.6.1"

  required_providers {
    snowflake = {
      source  = "Snowflake-Labs/snowflake"
      version = "0.80.0"
    }
  }
}

provider "snowflake" {
  account       = data.terraform_remote_state.snowflake_infrastructure.outputs.snowflake_account
  authenticator = "JWT"
}

provider "snowflake" {
  alias         = "account_admin"
  role          = "ACCOUNTADMIN"
  account       = data.terraform_remote_state.snowflake_infrastructure.outputs.snowflake_account
  authenticator = "JWT"
}

provider "snowflake" {
  alias         = "security_admin"
  role          = "SECURITYADMIN"
  account       = data.terraform_remote_state.snowflake_infrastructure.outputs.snowflake_account
  authenticator = "JWT"
}

provider "snowflake" {
  alias         = "sys_admin"
  role          = "SYSADMIN"
  account       = data.terraform_remote_state.snowflake_infrastructure.outputs.snowflake_account
  authenticator = "JWT"
}
```

### Expected Behavior

create external tables in snowflake

### Actual Behavior

╷
│ Error: Plugin did not respond
│
│ The plugin encountered an error, and failed to respond to the plugin6.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-snowflake_v0.80.0 plugin:

panic: interface conversion: interface {} is []interface {}, not []string

goroutine 66 [running]:
github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/resources.CreateExternalTable(0x0?, {0x10606ebc0?, 0x1400086a8f0?})
	github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/resources/external_table.go:181 +0xb84
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0x1060b2100?, {0x1060b2100?, 0x14000994de0?}, 0xd?, {0x10606ebc0?, 0x1400086a8f0?})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.30.0/helper/schema/resource.go:766 +0x134
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0x14000d14460, {0x1060b2100, 0x14000994de0}, 0x14000638750, 0x1400099c280, {0x10606ebc0, 0x1400086a8f0})
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.30.0/helper/schema/resource.go:909 +0x8a0
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0x14000c409a8, {0x1060b2100?, 0x14000994c00?}, 0x140009a6140)
	github.com/hashicorp/terraform-plugin-sdk/v2@v2.30.0/helper/schema/grpc_provider.go:1072 +0xb78
github.com/hashicorp/terraform-plugin-mux/tf5to6server.v5tov6Server.ApplyResourceChange({{0x1060c5920?, 0x14000c409a8?}}, {0x1060b2100, 0x14000994c00}, 0x0?)
	github.com/hashicorp/terraform-plugin-mux@v0.12.0/tf5to6server/tf5to6server.go:37 +0x58
github.com/hashicorp/terraform-plugin-mux/tf6muxserver.(*muxServer).ApplyResourceChange(0x1060b2058?, {0x1060b2100?, 0x140009948d0?}, 0x140009a60f0)
	github.com/hashicorp/terraform-plugin-mux@v0.12.0/tf6muxserver/mux_server_ApplyResourceChange.go:36 +0x188
github.com/hashicorp/terraform-plugin-go/tfprotov6/tf6server.(*server).ApplyResourceChange(0x14000bc6280, {0x1060b2100?, 0x14000994030?}, 0x1400099e000)
	github.com/hashicorp/terraform-plugin-go@v0.19.1/tfprotov6/tf6server/server.go:857 +0x3bc
github.com/hashicorp/terraform-plugin-go/tfprotov6/internal/tfplugin6._Provider_ApplyResourceChange_Handler({0x106017200?, 0x14000bc6280}, {0x1060b2100, 0x14000994030}, 0x1400099c000, 0x0)
	github.com/hashicorp/terraform-plugin-go@v0.19.1/tfprotov6/internal/tfplugin6/tfplugin6_grpc.pb.go:467 +0x170
google.golang.org/grpc.(*Server).processUnaryRPC(0x140001494a0, {0x1060b2100, 0x140010e8cf0}, {0x1060c1a20, 0x14000a8b040}, 0x14000a26000, 0x14000951410, 0x106d071b8, 0x0)
	google.golang.org/grpc@v1.59.0/server.go:1343 +0xc6c
google.golang.org/grpc.(*Server).handleStream(0x140001494a0, {0x1060c1a20, 0x14000a8b040}, 0x14000a26000)
	google.golang.org/grpc@v1.59.0/server.go:1737 +0x9f0
google.golang.org/grpc.(*Server).serveStreams.func1.1()
	google.golang.org/grpc@v1.59.0/server.go:986 +0x88
created by google.golang.org/grpc.(*Server).serveStreams.func1
	google.golang.org/grpc@v1.59.0/server.go:997 +0x174

Error: The terraform-provider-snowflake_v0.80.0 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.

### Steps to Reproduce

1. `terraform apply`

### How much impact is this issue causing?

High

**Comment by sfc-gh-jcieslak** [comment]
Hey @jacobcbeaudin. Thanks for reporting the issue.

The fix should be really simple. I'll push a pr for the fix today or tomorrow and it should be a part of the next release.

**Comment by oemergenc** [comment]
@sfc-gh-jcieslak Wow, thanks for fixing it so fast. It seems like the tests in the build are failing?
When will there be a next release?

**Comment by sfc-gh-jcieslak** [comment]
@oemergenc @jacobcbeaudin
It was released today as part of the 0.82.0 version. When you update please let me know if the issue has been fixed or just close the issue, Thanks.

**Comment by jacobcbeaudin** [comment]
> @oemergenc @jacobcbeaudin
> 
> It was released today as part of the 0.82.0 version. When you update please let me know if the issue has been fixed or just close the issue, Thanks.

0.82.0 fixed my issue. Thank you for the quick fix

**Comment by oemergenc** [comment]
@jacobcbeaudin @sfc-gh-jcieslak 
Unfortunately the issue does not seem to be fixed. Please see my comments here: 
https://github.com/Snowflake-Labs/terraform-provider-snowflake/issues/2293#issuecomment-1868969435
